### PR TITLE
Add sonar-project properties file

### PR DESCRIPTION
**Pull request by martinraynov**
# Description

Add sonarcloud config

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+sonar.projectVersion=1.0
+
+# Comma-separated paths to directories with sources (required)
+sonar.sources=.
+
+# Language
+sonar.language=python
+
+# Encoding of the source files
+sonar.sourceEncoding=UTF-8
```
Comment (pthegner) [inline sonar-project.properties:14]:
```suggestion
sonar.sourceEncoding=UTF-8

```

Comment (pthegner) [reply sonar-project.properties:14]: if possible, one line at the end of the file

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Required metadata
+sonar.organisationKey=canaltp
+sonar.projectKey=CanalTP_chaos
```
Comment (Fahmus) [inline sonar-project.properties:3]:
```suggestion
sonar.organisationKey=kisio
sonar.projectKey=Kisio_chaos
```

Comment (martinraynov) [reply sonar-project.properties:3]: The organisation is the name of our github organisation. So it has to be CanalTP

**Review by Fahmus — APPROVED** [review]

**Pull request closed without merging**